infrastructure/cdn-in-a-box/traffic_router/to-access.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
```

Comment:
don't want to have multiple copies of to-access.sh -- that's why all the containers have context set to the cdn-in-a-box level -- remove this and `ADD traffic_ops/to-access.sh` in the dockerfile